geoh5py/shared/conversion/geo_image.py

```diff
@@ -96,15 +108,15 @@ def add_rgb_data(cls, input_entity: GeoImage, output: Grid2D, name: str):
 
         output.add_data(
             data={
-                f"{name}_R": {
+                f"{name}_0R": {
```

Comment:
Why adding the int?

Reply:
because the data of objects are sorted by alphabetical order. 
I add it to preserve the order of the data for transform-backtransform consistency